index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" <link rel="stylesheet"
+        href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <div class="wraper">
+        <header>
+            <a href="#"><p>My Logo</p></a>
+            <nav><i class="fa-solid fa-bars"></i></nav>
```

Comment:
- [ ] As requested by the previous reviewer, kindly check and make sure that your logo and hamburger menu are aligned just as in the figma design.
- [ ] Also the color of the hamburger menu is different from that of the actual design. Kindly check and update it.


### Your design

![image](https://user-images.githubusercontent.com/65233860/220653352-fb457c07-b2fc-4cae-bcb4-ee798fa8e7b5.png)

### Figma design 

![image](https://user-images.githubusercontent.com/65233860/220653434-48748d97-4012-4726-a078-fc1218db17a9.png)